uitest/README.md

```diff
@@ -0,0 +1,44 @@
+## Dev Home UI Testing
+### How to run UI tests?
+1. From the command prompt, start the Windows Application Driver service:
```

Comment:
Can this be any command prompt or does it have to be the developer command prompt?

Reply:
Yes, any "shell" that can start the service process (e.g. Powershell, cmd, dev cmd, etc...)